stdlib/Distributed/src/cluster.jl

```diff
@@ -990,20 +990,20 @@ function _rmprocs(pids, waitfor)
 end
 
 
-struct ProcessExitedException <: Exception end
-
 """
-    ProcessExitedException()
+    ProcessExitedException(pid::Int)
 
 After a client Julia process has exited, further attempts to reference the dead child will
 throw this exception.
 """
-ProcessExitedException()
```

Comment:
Is this a stable API? We may need to define `ProcessExitedException() = ProcessExitedException(-1)`

Reply:
It seems this call to `ProcessExitedException()` was here to anchor the docstring above it.
Wouldn't it have been creating an object in global scope that was never used?
In any case, I removed it and placed the docstring before the struct definition.

When would you create a `ProcessExitedException` without the worker id? If there's no good reason for it, I think it would be better to not provide a no-args constructor.

Reply:
The point is that `ProcessExitedException()` works on Julia 1.0, and 1.1 but will throw an error after this PR.

Reply:
Oh right, for backwards compatibility for code outside of Base.
No-arg constructor `ProcessExitedException() = ProcessExitedException(-1)` added.